aten/src/ATen/native/native_functions.yaml

```diff
@@ -1734,6 +1734,9 @@
 - func: new_zeros(Tensor self, int[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
   variants: method
 
+- func: new_ones(Tensor self, int[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
```

Comment:
Let's add a test for the requires_grad flag: https://github.com/pytorch/pytorch/blob/eab59bae15bda508cdfbbc69a1a761b2b2c3780b/test/test_tensor_creation_ops.py#L1292. I don't know why new_ones isn't already in here